test/mic-check.html

```diff
@@ -0,0 +1,130 @@
+<html>
+  <head>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <button onclick="start()">Start!</button>
+    <div id="messages"></div>
+    🎤
+    <button onclick="recordingStart()">⏺</button>
+    <button onclick="recordingStop()">⏹</button>
+  </body>
+  <script type="module">
+    import {
+      Playthrough,
+      Speaker,
+      Microphone,
+      createPlaythroughToken,
+      createConversation,
+    } from "../dist/index.js";
+
+    import {
+      setGlobalBaseUrl,
+    } from "../dist/api.js";
+
+    // TEST PARAMETERS
+    const storyId = 2;
+    const version = -1;
+    const userToken = "eyXXX9.eyXXX.XX";
+    const startGraphReferenceId = "55f6a2ae-f526-4e50-b6ca-cfbf8cea4ffa";
+    const speechRecognitionStartEvent = {
+      service: "unified",
+      languageCode: "en-US",
+      returnRaw: false,
+      customServiceParameters: {}
+    };
+    const charismaUrl = "http://localhost:5050";
+    //
+
+    setGlobalBaseUrl(charismaUrl);
+
+    console.log();
+    const speaker = new Speaker();
+    const microphone = new Microphone();
+
+    let playthrough;
+    let conversation;
+
+    const messagesDiv = document.getElementById("messages");
+
+    window.start = async function start() {
+      const { token } = await createPlaythroughToken({ storyId, version, userToken });
+      console.log("New playthrough token:", token);
+      const { conversationUuid } = await createConversation(token);
+      console.log("New conversation ID:", conversationUuid);
+
+      playthrough = new Playthrough(token);
+
+      const timeout = 10000;
+
+      conversation = playthrough.joinConversation(conversationUuid);
+      conversation.on("message", (message) => {
+        microphone.resetTimeout(timeout);
+
+        const div = document.createElement("div");
+        div.classList.add("message", "character");
+        div.innerHTML = `<b>${message.message.character?.name || "???"}</b>: ${
+          message.message.text
+        }`;
+        messagesDiv.appendChild(div);
+      });
+
+      playthrough.on('speech-recognition-result', (speechResult) => {
+        if(speechResult.isFinal) {
```

Comment:
the lack of a space after `if` suggests `prettier` hasn't auto-formatted this doc, worth checking VS Code settings to see if it's running on `.html` files